### ISSUE TITLE
Create master list of operations at FOCI GRID stations

### ISSUE THREAD

**Issue by larogers123** [jesseflamb/Survey_Station_Selection#2]
I think we should un-tether ourselves from the 2019 survey plan and create fresh files that give the locations (FOCI GRID) for:

CalVET stations
CTD stations
Line 8 stations (maybe these can be within the CTD list)

This will help to make sure we prioritize those stations, and we can merge with our station list to determine operations at each station. This will also force us to be explicit about where those operations should occur rather than only relying on past practices.

**Comment by jesseflamb** [comment]
Agreed.  We should consult w/ Dave on whether these CalVET stations are now standard for this survey.  I believe some of the CTDs that we see are associated with the CalVETs, but don't know.  Line 8 is it's own thing.  The one thing, which we probably don't need to include, are the CTDs that were taken randomly for FastCAT calibration purposes.  

I will send an email to Dave inquiring about the CalVETs and associated CTDs and see if he has some standard assigned stations for this.
